msteams-platform/concepts/build-and-test/share-in-teams.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Share-in-Teams
+description: Learn to add the Share in Teams embedded on your personal app or tab
+ms.topic: reference
+ms.localizationpriority: medium
+keywords: Share Teams Share-in-Teams
+---
+# Create Share-in-Teams
+
+> [!NOTE]
+> Share-in-Teams is currently available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+Share-in-Teams allows you to share the content from personal app or tab to another user within Teams. You can select Share-in-Teams to launch the Share-in-Teams experience in a pop-up window. This pop-up window allows you to add another user or group or channel, notes and share the content.
```

Comment:
Not just another user. Other users or a Teams channel.

Reply:
Also, have we determined that "Share-in-Teams" is what we're calling this feature in public?

Reply:
'Share in Teams' is an internal term... For users and developers, it is still 'Share to Teams'